### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
Corrected some misspellings.
Tool used: https://github.com/crate-ci/typos (Try it on your blog!)

### CODE REVIEW

File: src/CountryCallingCode/CountryCallingCode.php [hunk 1]
```diff
@@ -80,7 +80,7 @@ enum CountryCallingCode: int
     case Germany_Federal_Republic_of                                                    = 49;
     case Ghana                                                                          = 233;
     case Gibraltar                                                                      = 350;
-    case Global_Mobile_Satelite_System_shared_code                                      = 881;
+    case Global_Mobile_Satellite_System_shared_code                                     = 881;
```
Comment (szepeviktor) [inline src/CountryCallingCode/CountryCallingCode.php:83]: Updated in https://www.itu.int/dms_pub/itu-t/opb/sp/T-SP-E.164D-2016-PDF-E.pdf page 8

Comment (szepeviktor) [reply src/CountryCallingCode/CountryCallingCode.php:83]: Viktor works for the Telecommunication Standardization Bureau 🙃

Comment (PrinsFrank) [reply src/CountryCallingCode/CountryCallingCode.php:83]: Can you put this change in a separate PR and target that at the 3.x branch? This is a breaking change if people already use this enum case somewhere hardcoded in their codebase. Can you also add it to the UPGRADING.md in that PR?

**Review by PrinsFrank — CHANGES_REQUESTED** [review]
Hey, thanks for the sponsorship and the PR! Apparently I'm not as good in English as I thought, that's quite some changes here. Can you split this into two as commented in the existing thread? I'll release the 3.x branch in a few months if there's some more breaking changes, the rest of these can be released with the next release. Thanks again!

File: src/CountryCallingCode/CountryCallingCode.php (hunk 1 above)
Comment (szepeviktor) [reply src/CountryCallingCode/CountryCallingCode.php]: Done!

**Review by PrinsFrank — APPROVED** [review]

**Comment by PrinsFrank** [comment]
Thanks!

**Comment by szepeviktor** [comment]
Glad to support your project.

**Pull request merged**